src/webapp/reports/validate-custom-forms/ValidateCustomFormsReport.tsx

```diff
@@ -0,0 +1,99 @@
+import { Typography, makeStyles } from "@material-ui/core";
+import _ from "lodash";
+import React, { useState } from "react";
+import i18n from "../../../locales";
+import { Spinner } from "../../components/objects-list/Spinner";
+import { Select, SelectOption } from "../../components/select/Select";
+import { useAppContext } from "../../contexts/app-context";
+
+const list = [{ text: "" }];
+
+const ValidateCustomFormsReport: React.FC = () => {
+    const [isLoading, setLoading] = useState(false);
+
+    const { compositionRoot, config } = useAppContext();
+    const OnModuleChange = async ({ value }: SelectOption) => {
```

Comment:
not sure what i should fix here

Reply:
@tokland sorry I think I forgot exactly how this could be changed. I just refactored the name and deleted an unnecessary function but I think I forgot something.